packages/puppeteer-extra-plugin-stealth/evasions/user-agent-override/index.js

```diff
@@ -129,14 +129,16 @@ class Plugin extends PuppeteerExtraPlugin {
       return greasedBrandVersionList
     }
 
+	const _padPlatformVersion = (version) => version+((version.split('.').length<=2)?'.0':'')
```

Comment:
format the code for better readability
```suggestion
    const _padPlatformVersion = (version) => version + ((version.split('.').length <= 2) ? '.0' : '')
```